internal/mode/static/state/dataplane/configuration.go

```diff
@@ -63,6 +60,7 @@ func BuildConfiguration(
 		CertBundles:           certBundles,
 		Telemetry:             telemetry,
 		BaseHTTPConfig:        baseHTTPConfig,
+		Logging:               logging,
```

Comment:
```suggestion
		Logging:               buildLogging(g),
```
Could probably simplify a couple of others as well while you're at it.